scripts/onebranch/post-build.ps1

```diff
@@ -12,21 +12,40 @@ Set-Location $scriptPath\..\..
 $OneBranchArch = $env:ONEBRANCH_ARCH
 $OneBranchConfig = $env:ONEBRANCH_CONFIG
 
+function Copy-BuildFolder {
+    param (
+        [Parameter(Mandatory=$true)]
+        [ValidateSet("Release", "Debug")]
+        [string]$Configuration
+    )
+
+    # Define the source folder and zip file path based on the configuration
+    $SourceFolder = ".\x64\$Configuration"
+    $ZipFile = ".\build\bin\x64_$Configuration\$Configuration.zip"
```

Comment:
It looks like the build artifact is formatted 'Build-x64-Release.zip'  - you might want to have the artifact here also produce this naming convention (see https://github.com/microsoft/ntosebpfext/releases/tag/Release-v0.1.0)

Reply:
fixed